setup.py

```diff
@@ -33,7 +33,8 @@
     author="EGA team",
     author_email="ega-helpdesk@ebi.ac.uk",
     python_requires=">=3.6",
-    install_requires=["requests", "tqdm", "htsget", "psutil"],
```

Comment:
Since `setup.py` doesn't specify versions, therefore the `pip install pyega3` command always downloads the latest version of these dependencies. However, our `requirements.txt` (see above) **does** specify versions. I decided to synchronize the `setup.py` and the `requirements.txt`, by copying the version numbers from the `requirements.txt` into the the `setup.py`.

Today, when someone runs the `pip install pyega3`, these are the dependencies, which are installed on their machine:

```
$ docker run --rm -it -v $PWD:/app -w /app python:3.6 bash       
root@14e6e8523d64:/app# pip install -e .                   
Downloading requests-2.26.0-py2.py3-none-any.whl (62 kB)
Downloading tqdm-4.62.2-py2.py3-none-any.whl (76 kB)
Downloading htsget-0.2.5.tar.gz (29 kB)
Downloading psutil-5.8.0-cp36-cp36m-manylinux2010_x86_64.whl (291 kB)
Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Downloading humanize-3.11.0-py3-none-any.whl (90 kB)
Requirement already satisfied: setuptools ... (57.5.0)
Downloading charset_normalizer-2.0.6-py3-none-any.whl (37 kB)
Downloading idna-3.2-py3-none-any.whl (59 kB)
Downloading urllib3-1.26.6-py2.py3-none-any.whl (138 kB)
Downloading certifi-2021.5.30-py2.py3-none-any.whl (145 kB)
```

You can see, that they are all different (=newer) from those versions, which we require/recommend in our `requirements.txt`.

I tested this change with all the supported Python versions (from 3.6 to 3.9) using the following commands and everything was fine and dandy:
```
local$ python -m coverage run -m pytest
local$ docker run --rm -it -v $PWD:/app -w /app python:3.6 bash

# Option #1:
docker$ pip install -e .
# Option #2:
docker$ python setup.py sdist
docker$ pip install dist/pyega3-*.tar.gz

docker$ pyega3 --test fetch EGAF00001753741
```

Of course, if this change is deemed/judged to be too much, then I can revert it easily.

By the way, I think we should just delete the `requirements.txt`, because the dependencies are automatically installed by the `pip3 install pyega3` command and we no longer recommend (in our README) pre-installing the dependencies using the `pip3 install -r requirements.txt` command.